src/core/Form/Select/MultiSelect/MultiSelectChipList/MultiSelectChipList.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { RefObject } from 'react';
+import { ChipList, Chip } from '../../../../Chip';
+
+import { MultiSelectData } from '../MultiSelect/MultiSelect';
+
+export interface MultiSelectChipListProps<T extends MultiSelectData> {
+  selectedItems: (T & MultiSelectData)[];
+  ariaChipActionLabel?: string;
+  sourceRef?: RefObject<any>;
+  onClick?: (item: T & MultiSelectData) => void;
+}
+
+export const MultiSelectChipList = <T extends MultiSelectData>(
+  props: MultiSelectChipListProps<T & MultiSelectData>,
+) => {
+  const { selectedItems, sourceRef, ariaChipActionLabel, onClick } = props;
+
+  const chipRefs = selectedItems
+    .filter((item) => !item.disabled)
+    .reduce(
+      (
+        arr: Array<React.RefObject<HTMLButtonElement>>,
+        _chip: T & MultiSelectData,
```

Comment:
Does the _chip serve a purpose here?

Reply:
Not sure, would need to test as this code is not from this PR.

Reply:
As discussed, should be left unchanged due to focus handling logic.